### PR TITLE
Fix broken link in nep-0046-sponsorship-guidelines.rst

### PR DESCRIPTION
**Pull request by morteza-im**
There was a broken link in [nep-0046-sponsorship-guidelines.rst](https://github.com/numpy/numpy/blob/main/doc/neps/nep-0046-sponsorship-guidelines.rst) file that I fixed it.

### CODE REVIEW

**Comment by mirzaim** [comment]
Fix #20919

File: doc/neps/nep-0046-sponsorship-guidelines.rst [hunk 1]
```diff
@@ -220,7 +220,8 @@ https://jupyter.org/about. Some subprojects have separate approaches, for
 example sponsors are listed (by using the `all-contributors
 <https://github.com/all-contributors/all-contributors>`__ bot) in the README for
 `jupyterlab-git <https://github.com/jupyterlab/jupyterlab-git>`__. For a recent
-discussion on that, see `here <jupyterlab-git acknowledgements discussion>`_.
+discussion on that, see
+`here <https://discourse.jupyter.org/t/ideas-for-recognizing-developer-contributions-by-companies-institutes/3178>`_.
```
Comment (mattip) [inline doc/neps/nep-0046-sponsorship-guidelines.rst:224]: Could you change the text to read "For a discussion from Jan 2020 on that, see"

**Comment by mattip** [comment]
Thanks @mirzaim

**Comment by InessaPawson** [comment]
Hi-five on merging your first pull request to NumPy, @mirzaim! We hope you stick around! Your choices aren’t limited to programming – you can review pull requests, help us stay on top of new and old  issues, develop educational material, work on our website, add or improve graphic design, create marketing materials, translate website content, write grant proposals, and help with other fundraising initiatives. For more info, check out: https://numpy.org/contribute
Also, consider joining our mailing list. This is a great way to connect with other cool people in our community and be part of important conversations that affect the development of NumPy: https://mail.python.org/mailman/listinfo/numpy-discussion

**Pull request merged**